Frontend/flight-search-amadeus/src/components/FlightCards/FlightCard.tsx

```diff
@@ -0,0 +1,54 @@
+export const FlightCard = () => {
```

Comment:
In future: please use object to mock data instead of direct data

Reply:
Will do, just was to put the base idea, I will work on the backend to send the mock data